contracts/tokens/Cruzo1155.sol

```diff
@@ -147,4 +149,36 @@ contract Cruzo1155 is Initializable, ERC1155URI {
     function setContractURI(string memory _newURI) external onlyOwner {
         contractURI = _newURI;
     }
+    function supportsInterface(bytes4 interfaceId)
+        public
+        view
+        override(ERC2981Upgradeable, ERC1155Upgradeable)
+        returns (bool)
+    {
+        return super.supportsInterface(interfaceId);
+    }
+
+    function setDefaultRoyaltyInfo(address _receiver, uint96 _royaltyFeesInBips)
+        public
+        onlyOwner
+    {
+        require(
+            _royaltyFeesInBips <= 5000,
+            "Royalty value must be between 0% and 50%"
+        );
+        _setDefaultRoyalty(_receiver, _royaltyFeesInBips);
+    }
+
+    function setTokenRoyalty(
```

Comment:
I think we need to invoke `setTokenRoyalty` also within `create` function. This way we don't have to run separate transaction after token creation.